rdt/transformers/categorical.py

```diff
@@ -223,28 +223,54 @@ class OneHotEncodingTransformer(BaseTransformer):
     dummy_na = None
     dummies = None
 
+    @staticmethod
+    def _prepare_data(data):
+        """Transform data to appropriate format.
+
+        If data is a valid list or a list of lists, transforms it into an np.array,
+        otherwise returns it.
+
+        Args:
+            data (pandas.Series, numpy.ndarray, list or list of lists):
+                Data to prepare.
+
+        Returns:
+            pandas.Series or numpy.ndarray
+        """
+        if isinstance(data, list):
+            data = np.array(data)
+            if len(data.shape) == 2:
+                if data.shape[1] != 1:
+                    raise ValueError("Unexpected format.")
+                data = data[:, 0]
```

Comment:
Can we add a couple of blank lines to separate the blocks after unindenting?

```python
if len(data.shape) == 2:
    if data.shape[1] != 1:
        raise ValueError("Unexpected format.")

    data = data[:, 0]

elif len(data.shape) > 2:
    raise ValueError("Unexpected format.")

return data
```

Reply:
Yes, this does look much more clean.